packages/ui/src/components/overlay-item-container/_overlay-item-container.scss

```diff
@@ -0,0 +1,10 @@
+.overlayItem {
```

Comment:
consider changing the name to overlayItemContainer to avoid confusion when looking at the html elements

Reply:
done